.env.example

```
@@ -0,0 +1,188 @@
+TOKEN2_DECIMALS_BASE_SEPOLIA=
+TOKEN3_DECIMALS_BASE_SEPOLIA=
+TOKEN4_DECIMALS_BASE_SEPOLIA=
+
+# ===================================================================
+# POOL CONFIGURATION
+# ===================================================================
+
+# Initial pool configuration
+POOL_INITIAL_FEE_BASE_SEPOLIA=
+POOL_INITIAL_TARGET_RATIO_BASE_SEPOLIA=
+POOL_TYPE_BASE_SEPOLIA=  # 0=STABLE, 1=STANDARD, 2=VOLATILE
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)